docs/Wiki/Executing_adb_commands.en.md

```diff
@@ -50,6 +54,23 @@ fun test() =
         // ....
 }
 ```
+This method passes each command to the java runtime which tokenizes it by whitespaces. It could be not ideal. It can't be used for the commands with
+the whitespaces in their arguments (e.g. `adb pull "path/with whitespace/file"`) and it doesn't allow piping (e.g. `cat $BUILD_DIR/report.txt | grep filte > filtered.txt`).
```

Comment:
"path/with whitespace/file" is a bit confusing.
maybe "path_with whitespase to_file" is better? but it is up to you.